lib/wallaroo/core/sink/tcp_sink/tcp_sink_builder.pony

```diff
@@ -115,12 +115,16 @@ class val TCPSinkBuilder
   fun apply(sink_name: String, event_log: EventLog,
     reporter: MetricsReporter iso, env: Env,
     barrier_coordinator: BarrierCoordinator, checkpoint_initiator: CheckpointInitiator,
-    recovering: Bool, worker_name_: WorkerName, auth_: AmbientAuth): Sink
+    recovering: Bool, app_name: String, worker_name: WorkerName,
+    auth_: AmbientAuth): Sink
   =>
     @printf[I32](("Connecting to sink at " + _host + ":" + _service + "\n")
       .cstring())
 
-    let id: RoutingId = RoutingIdGenerator()
+    let id: RoutingId = try DeterministicSourceIdGenerator(
```

Comment:
Wasn't this renamed so that it doesn't refer to `SourceId`?  Something like `RoutingIdFromStringGenerator`

Reply:
IIRC that change is what's causing the conflict with tcp_sink_builder.pony and perhaps the other conflict, too: the base of this branch is a bit old.